frontend/src/translations.json

```diff
@@ -57,7 +57,8 @@
       "starting_mileage": "Початкове значення вимірювального приладу",
       "unsynced_drives": "Не синхронізовані поїздки",
       "total_mileage": "Всього {total} км",
-      "unverified_drive": "Цей привід не перевірено, зв'яжіться з людиною з ПАУ, відповідальною за накопичувачі, щоб вирішити його"
+      "unverified_drive": "Цей привід не перевірено, зв'яжіться з людиною з ПАУ, відповідальною за накопичувачі, щоб вирішити його",
+      "verifired_drive": "Ця їзда перевірена"
```

Comment:
Two issues here (they apply to all languages):
* one message ends with a dot, the other doesn't contain it.
* there's a typo in verifi**r**ed_drive

Reply:
Fixed in #452